ocaml/utils/import_info.mli

```diff
@@ -12,7 +12,7 @@
 (*                                                                        *)
 (**************************************************************************)
 
-module CU = Compilation_unit
+module CU := Compilation_unit
 
 (* CR mshinwell: maybe there should be a phantom type allowing to distinguish
```

Comment:
This CR should be split and the first part removed (please change to `CR xclerc:`), since it's done (in a different way) by this patch.